src/core/core.controller.js

```diff
@@ -713,25 +713,22 @@ module.exports = function(Chart) {
 			// Built in Tooltips
 			if (tooltipsOptions.enabled || tooltipsOptions.custom) {
 				tooltip._active = me.tooltipActive;
+				tooltip._eventPosition = helpers.getRelativePosition(e, me.chart);
 			}
 
 			// Hover animations
 			if (!me.animating) {
 				// If entering, leaving, or changing elements, animate the change via pivot
-				if (!helpers.arrayEquals(me.active, me.lastActive) ||
```

Comment:
Does that mean the chart is redrawn at every mouse move? If so, that's not good :)